mungegithub/mungegithub.go

```diff
@@ -25,12 +25,13 @@ import (
 	"k8s.io/contrib/mungegithub/features"
 	github_util "k8s.io/contrib/mungegithub/github"
 	"k8s.io/contrib/mungegithub/mungers"
+	"k8s.io/contrib/mungegithub/mungers/mungerutil"
 	"k8s.io/contrib/mungegithub/reports"
 	utilflag "k8s.io/kubernetes/pkg/util/flag"
 
+	"github.com/foxish/viper"
```

Comment:
Since the toml library which was reverted has now been fixed, I'm ok with the fork, but can you take the lead of trying to update upstream viper again?